src/index.js

```diff
@@ -3,247 +3,267 @@
   Author Tobias Koppers @sokra
 */
 
-// TODO: temporarily disabled rules
-/* eslint-disable
-  no-undefined,
-  no-param-reassign,
```

Comment:
We shouldn't get rid of this. `no-param-reassign` is a bad rule, IMO. It makes us have to do confusing things like

```
+        compilation.plugin('build-module', (moduleArg) => {
            // to get detailed location info about errors
 -          module.useSourceMap = true;
 +          const moduleVar = moduleArg;
 +          moduleVar.useSourceMap = true;
          });
```